### PR TITLE
fix: replace native tail with fs-tail-stream

### LINKED ISSUE #111: Windows support
installed uing npm command, and when executin below command, getting below error? how to make it work in windows?

frontail C:\Users\Test\Downloads\apache-samples\apache-error_log\error_log\syslog

Error received:

events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: spawn tail ENOENT
    at _errnoException (util.js:1022:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:372:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:686:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3

### PR DESCRIPTION
**Pull request by mojoaxel**
fixes #111 (windows support)

### CODE REVIEW

**Comment by mthenw** [comment]
Wow, thanks for that PR. I will review it later today!

File: lib/tail.js [hunk 1]
```diff
@@ -23,30 +23,17 @@ function Tail(path, opts) {
...
-
-    tail.stderr.on('data', (data) => {
-      // If there is any important error then display it in the console. Tail will keep running.
-      // File can be truncated over network.
-      if (data.toString().indexOf('file truncated') === -1) {
-        console.error(data.toString());
-      }
+    const tail = tailStream.createReadStream(path.join(), {
+      encoding: 'utf8',
+      start: 80,
```
Comment (mthenw) [inline lib/tail.js]: I think start should be `options.buffer` here. `options.buffer` is what the user passes as a `-n` CLI param.

**Comment by mthenw** [comment]
Also, do you know if `fs-tail-stream` supports log rotation?

File: lib/tail.js (hunk 1 above)
Comment (mojoaxel) [reply lib/tail.js]: :confused: we have a problem here. the `-n` parameter is defined as lines but node's createReadStream uses bytes for the start parameter. I need to look into this...

**Comment by mojoaxel** [comment]
> Also, do you know if fs-tail-stream supports log rotation?

I don't think so :cry: 
This probably kills this PR

File: lib/tail.js (hunk 1 above)
Comment (mojoaxel) [reply lib/tail.js]: I tested it and changing `start` to `options.buffer` worked very good.

**Comment by mojoaxel** [comment]
Log-rotation is a problem! I tested it and the results are not pretty.
I don't think adding windows support in favor of log-rotation makes sense. :disappointed: 

Maybe I find another way to fix #111.

**Comment by mojoaxel** [comment]
@mthenw Could you please reopen this PR!

I've updated my branch to make the windows support additional to the existing `tail` support!
This way the existing functionality does not change and the windows support is just a fallback for systems without the `tail` command. Windows support is still missing support for log rotation. 

I think a lot of people would like to use frontail on windows even without log-rotation support!

**Comment by mthenw** [comment]
I'm not able to open this PR. Can you open a new one?

**Pull request closed without merging**